src/OSPSuite.Core/Domain/Services/SimModelBatch.cs

```diff
@@ -101,7 +111,8 @@ SimulationRunResults createSimulationRunResults(string errorFromException = null
          var warnings = WarningsFrom(_simModelSimulation);
          var error = errorFromException ?? (hasResults ? null : Error.SimulationDidNotProduceResults);
 
-         return new SimulationRunResults(success: error == null, warnings, getResults(), error);
+         var success = error == null;
+         return new SimulationRunResults(success, warnings, success ? getResults() : null, error);
```

Comment:
We created a secondary error message trying to get results in this case. A warning about time was generated and overwrote the error message from the simulation run

Reply:
Let's just simplify the way we return here. it's hard to understand when what is set